duckdb_engine/tests/test_pandas.py

```diff
@@ -117,3 +119,18 @@ def run_query(query: str, chunksize: Optional[int]) -> None:
         assert len(chunks[0]) == chunksize
         # Assert that the expected number of chunks was returned.
         assert (sample_rowcount / chunksize) == len(chunks)
+
+
+def test_read_sql_duckdb_table(tmp_path):
```

Comment:
I added this test which is partly duplicative with the test above, but written in a different style: I am first creating a table with duckdb, and then reading that with pandas/sqlalchemy (and then `read_sql("table_name")` actually seems to work ..). 

I am also not sure why this way, this test was failing before my patch, while the existing tests were not failing ..